site/pages/react/using-7702.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Using EIP-7702
+description: Learn how to use EIP-7702 to enable smart account features for EOAs
+---
+
+# Using EIP-7702
+
+## What is EIP-7702?
+
+[EIP-7702](https://eips.ethereum.org/EIPS/eip-7702) is an upcoming Ethereum upgrade set to launch in the coming months as part of the Pectra hardfork. Specifically, EIP-7702 enables Externally Owned Accounts (EOAs) to use smart contract account features. This upgrade marks a pivotal moment in Ethereum’s journey toward account abstraction, a breakthrough designed to make onchain experiences smoother, smarter, and more accessible. EIP-7702 doesn’t replace Ethereum’s existing AA standard, ERC-4337. EIP-7702 was designed to be complementary to ERC-4337 and make it easier for those account abstraction features to reach users.
```

Comment:
```suggestion
[EIP-7702](https://eips.ethereum.org/EIPS/eip-7702) is an upcoming upgrade set to launch as part of the Ethereum Pectra hardfork. Specifically, EIP-7702 enables Externally Owned Accounts (EOAs) to use smart contract account features.

This upgrade marks a pivotal moment in Ethereum’s journey toward account abstraction, a breakthrough designed to make onchain experiences smoother, smarter, and more accessible. EIP-7702 doesn’t replace Ethereum’s existing AA standard, ERC-4337. EIP-7702 was designed to be complementary to ERC-4337 and make it easier for those account abstraction features to reach users.
```

Reply:
For spacing/readability, and for us to not have to go back and edit the "in the coming months" part after we pass that milestone